### PR TITLE
synclogs db on staging

### PR DESCRIPTION
**Pull request by sravfeyn**
@snopoke https://github.com/dimagi/commcare-cloud/pull/1447#issuecomment-371179211

### CODE REVIEW

**Comment by sravfeyn** [comment]
Hi @emord can you review this and confirm if I can deploy this on staging so I can test [this](https://github.com/dimagi/commcare-hq/pull/19391#pullrequestreview-101840037) on staging first?

**Comment by sravfeyn** [comment]
@snopoke Can you review the last commit here (related to 9.6 upgrade).

File: environments/production/public.yml [hunk 1]
```diff
@@ -65,7 +65,7 @@ postgres_s3: True
 
 aws_region: 'us-east-1'
 
-
+postgresql_version: '9.4'
```
Comment (sravfeyn) [inline environments/production/public.yml]: FYI @NitigyaS I have updated this on all envs (by switching default to 9.6) you should remember update this after you upgrade postgres on prod.

Comment (NitigyaS) [reply environments/production/public.yml]: on enikshay also you need to set it 9.4 .

**Comment by snopoke** [comment]
Looks good once you address @NitigyaS comment.

**Pull request merged**